include/mbedtls/mbedtls_config.h

```diff
@@ -1931,7 +1931,8 @@
  * break backwards compatibility.
  *
  * \warning If you enable this option, you need to call `psa_crypto_init()`
- * before calling any function from the SSL/TLS, X.509 or PK modules.
+ * before calling any function from the SSL/TLS, X.509 or PK modules, except
+ * for the various mbedtls_xxx_init() functions which can be called at any time.
```

Comment:
Thanks! Forgot to mention, this warning is duplicated in `docs/use-psa-crypto.md`, can you update it there as well?